src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -18,6 +18,8 @@
 using System.Security.Cryptography.X509Certificates;
 using Microsoft.Win32.SafeHandles;
 
+using Internal.Runtime.CompilerServices;
```

Comment:
You will need to rework the makefiles to make this work. I think it may be easier to just not use Unsafe - change the code to:

`fixed (byte * p = &clientProto.DangerousGetPinnableReference()) outp = p;`

Reply:
Where else would this apply, or is this the only place?

Reply:
This is the only place in this delta.